torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -510,6 +576,50 @@ def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
             model = self._annotate_for_static_quantization_config(model)
         return model
 
+    def _annotate_by_single_config(
+        self,
+        model: torch.fx.GraphModule,
+        quantization_config: Optional[QuantizationConfig],
+        filter_fn: Optional[Callable[[Node], bool]] = None,
+    ) -> torch.fx.GraphModule:
+        """Select the annotator functions according to the `quantization_config` and apply."""
+
+        # <TODO> implement the support for None to be canceling out previous annotations
+        if quantization_config is None:
+            return model
+
+        if (
+            quantization_config.input_activation
+            and quantization_config.input_activation.is_dynamic
+        ):
+            annotators = DYNAMIC_ANNOTATORS
+        else:
+            annotators = STATIC_ANNOTATORS
```

Comment:
With the introduction of `STATIC_ANNOTATORS`, `DYNAMIC_ANNOTATORS` in this PR, we have 2 mechanism to do the annotation now:

- Get the `annotator_func` from `DYNAMIC_ANNOTATORS`/`STATIC_ANNOTATORS` and invoke it.
- Using the functions such as `_annotate_conv2d_fusion_pattern`, `_annotate_linear_fusion_pattern`

My feeling is same functionality can be achieved through either way, could you consolidate them to make the code clean and easy to debuig.